pkg/parser/sql_counter.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package parser
+
+// NumberOfStatements returns the number of SQL statement given a SQL program
+func NumberOfStatements(dialect, statements string) (int, error) {
```

Comment:
I am afraid we might not want to create a new file to host this function.  Also, I am not sure if we need this function; it's used only twice.  Adding this function creates more lines of code than not to have it.